android-components/plugins/dependencies/src/main/java/DependenciesPlugin.kt

```diff
@@ -50,6 +50,7 @@ object Versions {
 
     const val material = "1.9.0"
     const val ksp = "1.0.11"
+    val ksp_plugin = "$kotlin-$ksp"
```

Comment:
Drive-by: This is interesting.. does this mean we are now tied to specific versions of kotlin that have ksp plugin support? What happens if we need a patch version of `kotlin` and there is no `ksp` version?

Reply:
That should not happen, according to the docs:

> Usually a corresponding KSP release is available within a couple of days after a new Kotlin version is released, including the [pre-releases (Beta or RC)](https://kotlinlang.org/docs/eap.html).

source: https://kotlinlang.org/docs/ksp-faq.html#how-often-do-you-update-ksp